client/disputes/index.tsx

```diff
@@ -446,6 +447,7 @@ export const DisputesList = (): JSX.Element => {
 
 	return (
 		<Page>
+			<MerchantFeedbackPrompt />
```

Comment:
If we're being selective about which pages we put this prompt on, I wonder if we should avoid disputes.

Reply:
🤔

Reply:
I'll leave this decision to another project issue, which I'll set as [a draft on our project board](https://github.com/orgs/Automattic/projects/578/views/1?visibleFields=%5B%22Title%22%2C32157446%2C68744602%2C%22Status%22%2C%22Assignees%22%2C%22Labels%22%2C%22Repository%22%2C46101517%2C%22Linked+pull+requests%22%2C%22Milestone%22%5D&sliceBy%5Bvalue%5D=Prompt+merchants+to+provide+feedback&pane=issue&itemId=100216471).